src/Tests/Microsoft.NET.TestFramework/Microsoft.NET.TestFramework.csproj

```diff
@@ -20,6 +20,13 @@
     <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Clever way of passing through the newtonsoft version the SDK builds with to the tests. I can't think of a reason why we'd want this to be different between the tests and product. @dsplaisted , any concerns with this approach?